book/src/server_configuration.md

```diff
@@ -27,8 +27,12 @@ text=You MUST set the `domain` name correctly, aligned with your `origin`, else
 You should test your configuration is valid before you proceed.
 
 ```bash
+# Docker:
```

Comment:
Please remove non-container references from the changes here, we only _support_ containerised servers. Folks can run whatever they like, but from a docs/support perspective, that's where we prefer things.

Reply:
Agreed, the server packages are not an 'upstream' thing. We will look to support packages of the *client* components, especially pam and nss, but containers are the preference for the server.